compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -81,6 +81,8 @@ static void PrintMethodFields(
   (*vars)["service_name"] = service->name();
   for (int i = 0; i < service->method_count(); ++i) {
     const MethodDescriptor* method = service->method(i);
+    (*vars)["i"] = to_string(2 * i);
```

Comment:
consider a better name? I'm fine with short names, but since we are inside the for loop, `i` is a little confusion.